chat/src/main/java/net/md_5/bungee/chat/ComponentSerializer.java

```diff
@@ -163,6 +164,16 @@ public static String toString(Object object)
         return gson.toJson( object );
     }
 
+    public static String toString(Content content)
```

Comment:
What's the reason for these being added?

Reply:
During work on this, I noticed #3629 and wanted to remove any invocations of `toString(Object)`, and here I jsut forgot to mention it.
I would further suggest to deprecate the toString(Object) method here.

Reply:
Added deprecation